packages/docgen/src/test/formatter-markdown.js

```diff
@@ -38,36 +39,4 @@ describe( 'Formatter', () => {
 			'# API docs\n\n<a name="myDeclaration" href="#myDeclaration">#</a> **myDeclaration**\n\nMy declaration example.\n\n*Parameters*\n\n- *firstParam* `number`: First declaration parameter.\n\n*Returns*\n\n- `number`: The result of the declaration.\n'
 		);
 	} );
-
-	it( 'handles unknown types from parse error', () => {
```

Comment:
What happens when there's a parser error using the new setup?

Reply:
I saw there's a related test removed in `get-intermediate-representation` as well.

Reply:
I'll do some digging into the library to check and see how to trigger an error. This new library is pretty "simple" in that it doesn't seem to have many expectations about how the input is formatted except for the additional `/*` and `/` that needed to be prepended/appended to the comment input string.

Reply:
Same as below: I guess it makes sense to remove this. As I understand it, comment-parser does actually not parse the type, just passes it through.